x/ibc/02-client/keeper/client.go

```diff
@@ -72,13 +86,18 @@ func (k Keeper) UpdateClient(ctx sdk.Context, clientID string, header exported.H
 	}
 
 	var (
-		consensusState exported.ConsensusState
-		err            error
+		consensusState      exported.ConsensusState
+		err                 error
+		oldHeaderNextValSet *tmtypes.ValidatorSet // TODO: should I get this from the msg or internal query?
```

Comment:
cc: @cwgoes

Reply:
Ideally we get this from the lite client store. Maybe it does need to be passed into the keeper tho.

Reply:
> lite client store

we don't have that structure atm tho

Reply:
Talked to the TM team about this this morning. Sounds like we need to make some changes to enable this.